storage-proto/src/convert.rs

```diff
@@ -728,11 +737,28 @@ impl TryFrom<tx_by_addr::TransactionError> for TransactionError {
     fn try_from(transaction_error: tx_by_addr::TransactionError) -> Result<Self, Self::Error> {
         if transaction_error.transaction_error == 8 {
             if let Some(instruction_error) = transaction_error.instruction_error {
+                // Decode the index.
+                //
+                // Index values are 32-bit integers of the form:
+                // TTTTTTTT IIIIIIII xxxxxxxx xxxxxxxx
```

Comment:
This is already a `uint32`, so we can fit more u8 data in here, rather than to create a new column.